cmd/spire-server/cli/entry/delete.go

```diff
@@ -145,7 +145,11 @@ func (c *deleteCommand) prettyPrintDelete(env *commoncli.Env, results ...any) er
 		return errors.New("failed to delete one or more entries")
 	}
 
-	env.Printf("Deleted %d entries successfully, but failed to delete %d entries", len(succeeded), len(failed))
+	if len(failed) > 0 {
```

Comment:
There's already a a check like this just before this, which ends up returning so we never enter this if block. Could you combine the two together?

Reply:
okay

Reply:
upated the PR